ui/components/component-library/popover/README.mdx

```diff
@@ -11,8 +11,6 @@ Popover is an overlay that appears by the trigger used for menus, additional con
 
 ## Props
 
-The `Popover` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) component props
```

Comment:
Through TypesScript we can now view all props in the prop table including the Box component style utility props so this line is redundant